extensions/functions_arithmetic_decimal.yaml

```diff
@@ -174,3 +174,95 @@ aggregate_functions:
         decomposable: MANY
         intermediate: "DECIMAL<38,S>"
         return: "DECIMAL<38,S>"
+
+  -
+    name: "power"
```

Comment:
This the aggregate functions section.  These definitions should be placed in the scalar functions section.

Reply:
fixed it